.github/workflows/on-chain-upgrade.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     env:
       GENESIS_DESMOS_VERSION: "v4.1.0"
       GENESIS_URL: "https://raw.githubusercontent.com/RiccardoM/desmos-states/master/morpheus-apollo-2-6608740.json"
```

Comment:
It's fine to keep these data here since the only thing that's contained inside the `v4.3.0` and `v4.4.1` upgrades are incremental migrations. If the migration from `v4.1.0` up until `v4.4.1` is successful, then it will also be the same for the one from `v4.3.0` to `v4.4.1`.